lib/ansible/modules/web_infrastructure/jira.py

```diff
@@ -263,7 +263,8 @@ def request(url, user, passwd, timeout, data=None, method=None):
     body = response.read()
 
     if body:
-        return json.loads(body)
+        return json.loads(to_text(body, errors='surrogate_or_strict'))
+        
```

Comment:
You have trailing whitespace over this line. Either remove trailing whitespaces or remove this line.